.github/workflows/s3-rollout.yml

```
@@ -0,0 +1,90 @@
+---
+name: Rollout S3 buckets
+on:  # yamllint disable-line rule:truthy
+  workflow_call:
+    inputs:
+      # Scaleway information
+      s3-region:
```

No need to prefix these with `s3`, we don't do it for the other (internal) workflows. This is a workflow for s3 stuff, so it's obvious from that.
```suggestion
      region:
```

---

Same for provider then?